build.gradle

```diff
@@ -42,24 +42,31 @@ artifactory {
 
 jenkinsPlugin {
     pluginId = 'synopsys-security-scan-plugin'
-    jenkinsVersion = '2.387.3'
+    jenkinsVersion = '2.377'
     displayName = 'Synopsys Security Scan Plugin'
 }
 
 dependencies {
+    implementation 'org.jenkins-ci.plugins:cloudbees-bitbucket-branch-source:805.v7f97d29dc0f5'
+    implementation 'org.jenkins-ci.plugins:matrix-project:785.v06b_7f47b_c631'
+    implementation 'org.jenkins-ci.plugins:cloudbees-folder:6.15'
     annotationProcessor 'com.synopsys.integration:jenkins-annotation-processor:0.0.7'
 
+    implementation 'org.jenkins-ci.plugins:credentials:1139.veb_9579fca_33b_'
+    implementation 'com.synopsys.integration:blackduck-common:66.1.4'
     implementation 'com.synopsys.integration:jenkins-common:0.5.5'
-    implementation 'org.jenkins-ci.plugins:cloudbees-bitbucket-branch-source:809.vc1d904b_30426'
+    implementation 'org.jvnet.localizer:localizer:1.31'
     implementation 'org.jsoup:jsoup:1.16.1'
 
-    workflowJobDslApi 'org.jenkins-ci.plugins:job-dsl:1.84'
-    workflowJobApi 'org.jenkins-ci.plugins.workflow:workflow-job:1316.vd2290d3341a_f'
-    workflowCpsApi 'org.jenkins-ci.plugins.workflow:workflow-cps:3697.vb_470e454c232'
-    workflowStepApiApi 'org.jenkins-ci.plugins.workflow:workflow-step-api:625.vd896b_f445a_f8'
+    implementation 'org.jenkins-ci.plugins:plain-credentials:1.8'
```

Comment:
we may not need this if we remove `org.jenkins-ci.plugins:credentials:1139.veb_9579fca_33b_` this dependency